src/components/button/secondaryButton.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by jonlazarini on 14/02/17.
+ */
+import React from 'react';
+// import Button from './index.jsx';
+import PrimaryButton from './primaryButton.jsx';
+import styled from 'styled-components';
+
+
+// Style wrapper for default buttons - Composition
+const SecondaryWrapper = (Comp) => styled(Comp)`
+  margin-top: 0.2em;
+  padding: 0.6em 1.6em;
+  border: 2px solid black;
+  font-size: 0.8em;
+  font-weight: 600;
+  text-transform: lowercase;
+  background: black;
+  color: white;
+  &:hover {
+    background: white;
+    color: black;
+    cursor: pointer;
+    }
+  `;
+
+const SecondaryButton = SecondaryWrapper(PrimaryButton);
```

Comment:
You can either modify the style on the primary button which should be named base as it was the intent rather than primary and compose new style(s) from it by extending or overriding the style if necessary.
We could pass the Button instead and was declared this for a concern of not repeating myself for the button base styles.
Renaming the button `base` would make more sense perhaps?